DSCResources/MSFT_xWebsite/MSFT_xWebsite.schema.mof

```diff
@@ -1,27 +1,28 @@
-[ClassVersion("1.0.0")]
-Class MSFT_xWebBindingInformation
+[ClassVersion("1.1.0")]
```

Comment:
Both classes were modified, so I decided it was a good idea to increment their version numbers.